.github/workflows/ci_front_build_android.yml

```diff
@@ -4,14 +4,14 @@ on:
   push:
     branches: [ "master" ]
   pull_request:
-    branches: [ "master" ]
+    branches: [ "master", "dev" ]
```

Comment:
Why is it changed in this PR?

Reply:
@bjadczak changed it in order to trigger builds within this PR